lib/flowViewport/sceneIndex/fvpPruneTexturesSceneIndex.cpp

```diff
@@ -60,16 +60,17 @@ _PruneTexturesFromMatNetwork(
 // static
 PruneTexturesSceneIndexRefPtr
 PruneTexturesSceneIndex::New(
-    const HdSceneIndexBaseRefPtr &inputSceneIndex)
+    const HdSceneIndexBaseRefPtr &inputSceneIndex,
+    bool pruneTextures)
```

Comment:
Initial setting was don't prune (keep textures), which is wrong, so allow passing in setting on scene index tree creation.